src/Tribe/Repository.php

```diff
@@ -1609,6 +1609,60 @@ public function where_meta_related_by( $meta_keys, $compare, $field = null, $val
 
 		return $this;
 	}
+	/**
+	 * Filters the query to only return posts that are related, via a meta key, to posts
+	 * that satisfy a condition.
+	 *
+	 * @param string|array $meta_keys   One or more `meta_keys` relating the queried post type(s)
+	 *                                  to another post type.
+	 * @param string       $compare     The SQL comparison operator.
+	 * @param string       $meta_field  One (a column in the `postmeta` table) that should match
+	 *                                  the comparison criteria; required if the comparison operator is not `EXISTS` or
+	 *                                  `NOT EXISTS`.
+	 * @param string|array $meta_values One or more values the post field(s) should be compared to;
+	 *                                  required if the comparison operator is not `EXISTS` or `NOT EXISTS`.
+	 *
+	 * @return $this
+	 * @throws Tribe__Repository__Usage_Error If the comparison operator requires
```

Comment:
I think that should come before the `@params`, right after the `@since`.

Reply:
Was following the format of the function above it - but happy to change